maintainers/maintainer-list.nix

```diff
@@ -11639,6 +11639,12 @@
     email = "4ZKTUB6TEP74PYJOPWIR013S2AV29YUBW5F9ZH2F4D5UMJUJ6S@hash.domains";
     name = "Thomas Strobel";
   };
+  tshaynik = {
```

Comment:
Please put this into a separate commit with the message ``maintainers: add NAME`` and sort it alphabetically.